Updated CI test to use pylint 2.2.2

joachimmetz opened the pull request.

berggren commented:
@joachimmetz tests are failing with:
find: ‘tests’: No such file or directory

I don't use a single test directory in TS. Before this, nosetest discovered all tests. How are we doing test file discovery in this setup?

berggren commented:
@joachimmetz #831 is merged now.

joachimmetz commented on config/travis/install.sh:
```diff
@@ -0,0 +1,25 @@
+set -e;
+if test ${TRAVIS_OS_NAME} = "linux" && test ${TARGET} != "jenkins";
+then
+	sudo rm -f /etc/apt/sources.list.d/travis_ci_zeromq3-source.list;
```
yes this is to work around some older issue. I'll remove it and make a not to remove it entirely if no longer needed.

joachimmetz replied:
https://github.com/log2timeline/l2tdevtools/issues/638

joachimmetz commented:
> @joachimmetz #831 is merged now.

thx, I've changed pylint tests, path are hard coded in the runtests.sh script
removed the work-around and small changes to .pylintrc.

Let's see if the tests pass, I assume a couple of more iterations due to linter changes

joachimmetz commented:
@berggren changes made with additional changes to get linter passing
